src/libraries/System.Globalization/tests/Invariant/InvariantMode.cs

```diff
@@ -13,6 +14,23 @@ namespace System.Globalization.Tests
 {
     public class InvariantModeTests
     {
+        private static bool PredefinedCulturesOnlyIsDisabled => !PredefinedCulturesOnly();
```

Comment:
Did you intend for this property to call `PredefinedCulturesOnly()` every time it is invoked? Or did you want it to be a static field that is initialized once?

Reply:
Although it is not a big deal to have this property called a few times, but it is not required to do so. I'll convert it to a static property.